File: CS/EmptyWinApp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraPivotGrid;
using DevExpress.XtraPivotGrid.Data;

namespace EmptyWinApp {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        private void Form1_Load(object sender, EventArgs e) {
            this.productReportsTableAdapter.Fill(this.productReports._ProductReports);
        }
        readonly object dummyItem = new object();
        private void pivotGridControl1_CustomFilterPopupItems(object sender,
                                            PivotCustomFilterPopupItemsEventArgs e) {
            if(object.ReferenceEquals(e.Field, fieldCategoryName)) {
                for(int i = e.Items.Count - 1; i >= 0; i--) {
                    if(object.Equals(e.Items[i].Value, "Beverages")) {
                        e.Items.RemoveAt(i);
                        break;
                    }
                }
                e.Items.Insert(0, new PivotGridFilterItem(dummyItem,
                                                          "Dummy Item",
                                                          e.Field.FilterValues.Contains(dummyItem)));
            }
        }
    }
}

File: VB/EmptyWinApp/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraPivotGrid
Imports DevExpress.XtraPivotGrid.Data

Namespace EmptyWinApp
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.productReportsTableAdapter.Fill(Me.productReports._ProductReports)
		End Sub
		Private ReadOnly dummyItem As Object = New Object()
		Private Sub pivotGridControl1_CustomFilterPopupItems(ByVal sender As Object, _
					ByVal e As PivotCustomFilterPopupItemsEventArgs) _
					Handles pivotGridControl1.CustomFilterPopupItems
			If Object.ReferenceEquals(e.Field, fieldCategoryName) Then
				For i As Integer = e.Items.Count - 1 To 0 Step -1
					If Object.Equals(e.Items(i).Value, "Beverages") Then
						e.Items.RemoveAt(i)
						Exit For
					End If
				Next i
				e.Items.Insert(0, New PivotGridFilterItem(dummyItem, _
								"Dummy Item", _
								e.Field.FilterValues.Contains(dummyItem)))
			End If
		End Sub
	End Class
End Namespace